.yarnrc.yml

```diff
@@ -1,11 +1,8 @@
 logFilters:
-  # react-virtualized thinks it will only work with react 16.x,
```

Comment:
Are the comments here no longer correct? If they are correct, I think it would be good to keep them.

Reply:
yarn removes these comments when it modifies this file (for example, when using `yarn set version`). I forgot to re-add them this time. I can do that, but they're likely to get deleted at some point since yarn doesn't seem to want comments here.